### DISCUSSION TITLE
Flagged account

### DISCUSSION THREAD

**Discussion by nikhilkumarApTSi** [community/community#129543] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I granted access to the repositories in a free account that I maintained to other github users. When I added them as members and they tried to accept the invite they got an account flagged error. I recommended that they start a ticket and get guidance from github support to free up their accounts and get access to the repos, but I am struggling with why they were flagged and how to get them unlocked and have access to my account. Since the flagging I had them make sure that they had 2 factor authentication setup and the account also have their verification email setup. Is there something else that they should be doing? Should I also submit a ticket?

**Comment by ebndev** [comment]
Hi @nikhilkumarApTSi,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

I think you should submit one as well, since they are getting flagged from a repository on your account.

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
